lua/entities/gmod_wire_gimbal.lua

```diff
@@ -57,7 +62,12 @@ function ENT:Think()
 		elseif self.TargetAng then
 			ang = self.TargetAng
 		end
-		if ang then self:SetAngles(ang + Angle(90,0,0)) end
+		if ang then
+			local m = Matrix()
+			m:SetAngles(ang)
+			m = m * self.TargetAngOffset
```

Comment:
http://wiki.garrysmod.com/page/VMatrix/Rotate

Reply:
SetAngles is faster

Reply:
Also TargetAngOffset is a matrix

Reply:
> SetAngles is faster

Do you have a source for this or any benchmarks? If it is faster I'm happy for it to stay, with with a comment explaining that we're doing a non-obvious thing here because it's faster than the obvious thing. But without measurements it's a meaningless claim.

> Also TargetAngOffset is a matrix

Yes, a matrix that's created from an angle, and only converted to a matrix for the purpose of doing this multiplication here. My suggestion is that it should be left as an angle.

Reply:
Matrix multiplication is faster. I can benchmark it later.

Reply:
I judge speed by flops, and I'm basing how many flops each does based on the algorithm, since I know how each mathematically works.

Reply:
There is an offchance one was implemented poorly though, which a bench mark would reveal.